indico/htdocs/sass/partials/_infogrids.scss

```diff
@@ -133,3 +133,61 @@
         top: 0;
     }
 }
+
+ul.steps {
+    list-style-type: none;
+    padding: 0;
+    color: $light-black;
+
+    li {
+        flex-grow: 1;
+        flex-basis: 0;
+
+        &.completed {
+            .timestamp {
+                border-top-color: $indico-blue;
+
+                &:before {
```

Comment:
According to our sass-lint and [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements), pseudo-elements must start with double colons. So `&::before` would make sass-lint happier.